shared/chat/inbox/index.desktop.tsx

```diff
@@ -38,13 +38,146 @@ const FakeRow = ({idx}: {idx: number}) => (
 
 const FakeRemovingRow = () => <Kb.Box2 direction="horizontal" style={styles.fakeRemovingRow} />
 
-const dragKey = '__keybase_inbox'
+const dragKey = 'application/keybase_inbox'
+
+const DragLine = (p: {
+  scrollDiv: React.RefObject<HTMLDivElement>
+  inboxNumSmallRows: number
+  showButton: boolean
+  smallTeamsExpanded: boolean
+  toggleSmallTeamsExpanded: () => void
+  setInboxNumSmallRows: (n: number) => void
+  style: Object
+  rows: T.Chat.ChatInboxRowItem[]
+}) => {
+  const {inboxNumSmallRows, showButton, style, scrollDiv} = p
+  const {smallTeamsExpanded, toggleSmallTeamsExpanded, rows, setInboxNumSmallRows} = p
+  const [dragY, setDragY] = React.useState(-1)
+  const deltaNewSmallRows = React.useCallback(() => {
+    if (dragY === -1) {
+      return 0
+    }
+    return Math.max(0, Math.floor(dragY / smallRowHeight)) - inboxNumSmallRows
+  }, [dragY, inboxNumSmallRows])
+
+  const newSmallRows = deltaNewSmallRows()
+  let expandingRows: Array<string> = []
+  let removingRows: Array<string> = []
+  if (newSmallRows === 0) {
+  } else if (newSmallRows > 0) {
+    expandingRows = new Array<string>(newSmallRows).fill('')
+  } else {
+    removingRows = new Array<string>(-newSmallRows).fill('')
+  }
+
+  const throttledDragY = C.useThrottledCallback(setDragY, 100)
+
+  const onDragOver = React.useCallback(
```

Comment:
before this was throttled but doing this you can lose the `.dataTransfer` value somehow, if we don't then its always there so instead we just throttle updating the `dragY`